CHANGES.rst

```diff
@@ -107,6 +107,8 @@ API changes
 
 - ``astropy.wcs``
 
+  - Improved documentation of WCS.calc_footprint [#3551]
```

Comment:
Minor documentation updates don't generally warrant a changelog entry.  It's more for "Is this a change in behavior a user might notice" (sometimes that's true for docs in cases where the docs were wrong, but that's not really true here).  Thanks for the diligence nonetheless!  This subtlety is not documented anywhere so it was still a good guess to go ahead and include it.